src/linttest/linttest.go

```diff
@@ -213,6 +223,8 @@ func (s *Suite) RunLinter() []*linter.Report {
 		}
 	}
 
+	linter.ApplyQuickFixes = s.ApplyQuickFixes
```

Comment:
In the current architecture, you don't need a suite field to set the global variable.
I would set the `linter.ApplyQuickFixes` inside the quickfixes test.